packages/plexus/src/zoom/MiniMap.tsx

```diff
@@ -81,10 +81,10 @@ function getClassNames(props: TProps) {
   };
 }
 
-export function MiniMap(props: TProps) {
-  const css = getClassNames(props);
-  const mapSize = getMapSize(props);
-  const activeXform = getViewTransform(props, mapSize);
+export function MiniMap({ className = '', classNamePrefix = 'plexus', ...props }: TProps) {
+  const css = getClassNames({ className, classNamePrefix, ...props });
```

Comment:
this function does not use `props`. You can change it to take `className, classNamePrefix` explicitly.